image/image.go

```diff
@@ -787,3 +787,42 @@ func copyLocalSnapFile(snapPath, targetDir string, info *snap.Info) (dstPath str
 	dst := filepath.Join(targetDir, filepath.Base(info.MountFile()))
 	return dst, osutil.CopyFile(snapPath, dst, 0)
 }
+
+func ValidateSeed(seedFile string) error {
+	seed, err := snap.ReadSeedYaml(seedFile)
+	if err != nil {
+		return err
+	}
+
```

Comment:
I would also add a check that either "core" or "snapd" are present.  To check more than that we would need to also read the the model assertion.

Reply:
Great point, thank you, I added this now.